platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -510,6 +547,26 @@ namespace platoon_strategic_ihp
             */
             bool isVehicleRightBehind(double downtrack);
             
+            /**
+            * \brief Function to determine if the target platoon leader is in an adjacent lane (used for cut-in from front) 
+            *
+            * \param type boolean
+            *
+            * \return if target vehicle in front
+            */
+            bool isVehicleNextLane(double frontVehicleDtd, double rearVehicleDtd, double frontVehicleCtD);
+
+            /**
+            * \brief Function to find the starting and ending lanelet ID for lane change in a two-lane scenario (used for cut-in from front) 
+            * 
+            * Note: This is a temporary function for internal test only. The scenario is not genrealized and further development is needed.
```

Comment:
For temp code, please use a TODO comment so that it's easy to search for.  These things have a way of being forgotten about.

Reply:
Added TODO mark.